crank/tests/test_stack.py

```diff
@@ -0,0 +1,188 @@
+"""
+Unit and regression test for the crank package.
+"""
+
+import pytest
+import os, sys, subprocess, filecmp, shutil, json
+import numpy as np
+from crank.DihedralScanner import DihedralScanner, Molecule
+from crank.QMEngine import QMEngine
+from crank.PriorityQueue import PriorityQueue
+import geometric
+
+try:
+    import qcengine
+    import psi4
+except:
+    pass
+
+bohr2ang = 0.529177210
```

Comment:
I would pull this from Crank itself or geomeTRIC instead of building your own.

Reply:
Fixed.